src/style.css

```diff
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("truetype");
+  font-weight: 400;
+  font-display: swap;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+  font-weight: 300;
+  font-display: swap;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+  font-weight: 900;
+  font-display: swap;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+form label {
+  position: relative;
+}
+
+form label:before {
+  content: "";
+  position: absolute;
+  left: 26px;
+  top: 2px;
+  background-image: url(./images/Search.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  z-index: 1;
+}
+
+form.small-form label:before {
+  left: 13px;
+  top: 5px;
+  width: 11px;
+  height: 11px;
+}
+
+form {
+  width: 100%;
+  margin-top: 20px;
+}
+
+.big-form input {
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  width: 100%;
+  height: 70px;
+  font-family: inherit;
+  transition: 0.4s ease-in-out;
+}
+
+.small-form input {
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  height: 42px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  transition: 0.4s ease-in-out;
+  font-family: inherit;
+}
```

Comment:
You can avoid repeating properties if you follow this part from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

![image](https://user-images.githubusercontent.com/80700247/227747065-52f66cd9-8216-4a2f-819a-2e52150b7784.png)

Use this everywhere it's possible